debian/pbspro-server.install

```diff
@@ -0,0 +1,308 @@
+etc/pbs_db_schema.sql /opt/pbs/etc
```

Comment:
@subhasisb, rpm concentrates most of it's build process around the spec file and rpm macros. Debian on the other hand, distributes a lot among different files. There is near to nothing common between them as far as I have understood.